Monika After Story/game/script-topics.rpy

```diff
@@ -7010,3 +7010,39 @@ label monika_dating_startdate_confirm(first_sesh_raw):
 
     # default action is to loop here
     jump .loopstart
+
+init 5 python:
+     addEvent(Event(persistent.event_database, eventlabel="monika_first_sight_love", category=["life", "romance"], prompt="Love at first sight", random=True))
+
+label monika_first_sight_love:
+    m 1d "Have you thought about the concept of 'love at first sight'?"
+    m 3d "Like, seeing someone for the first time, and instantly know they will be the love of life."
```

Comment:
"love of life?"
Missing the word "your".
It technically can do without the question mark, if considered nothing but an elaboration upon the previous question, or it can also have a question mark depending on the tone you want to set.